src/modules/navigation/DrivesNavItem.jsx

```diff
@@ -1,32 +1,86 @@
 import React from 'react'
 
 import { useQuery } from 'cozy-client'
+import flag from 'cozy-flags'
+import { NavDesktopLimiter } from 'cozy-ui/transpiled/react/Nav'
 import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
 
+import { ROOT_DIR_ID } from 'constants/config'
+import { CozyFile } from 'models/index'
 import { NavItem } from 'modules/navigation/NavItem'
-import { SharedDrives } from 'modules/views/Folder/SharedDrives'
 import { buildSharedDrivesQuery } from 'modules/views/Folder/queries/fetchSharedDrives'
 
 const DrivesNavItem = ({ clickState }) => {
   const { isMobile } = useBreakpoints()
-  const sharedDrivesQuery = buildSharedDrivesQuery()
+  const { t } = useI18n()
+
+  const sharedDrivesQuery = buildSharedDrivesQuery({
+    sortAttribute: 'name',
+    sortOrder: 'asc'
+  })
   const { data } = useQuery(
     sharedDrivesQuery.definition,
     sharedDrivesQuery.options
   )
 
-  const label = data?.length > 0 ? 'drives' : 'drive'
+  const hasMultipleDrive = data?.length > 0
+  const showNextcloud = flag('drive.show-nextcloud-dev') == true
```

Comment:
Yes if you have shortcut into the shared drives folder it will display them there. In the future, it will have also folder as the real version of shared drives